install.sh

```diff
@@ -610,13 +614,13 @@ install_chaos_mesh() {
     local runtime=$7
     local k3s=$8
     local version=$9
-    local timezone=$10
-
+    local timezone=${10}
+    local docker_registry=${11}
```

Comment:
```
    local timezone=$10

    local docker_registry=$11
```
Keep a uniform format

Reply:
We must add `{}` when more than 10 parameters in shell.